Dockerfile

```diff
@@ -1,16 +1,17 @@
+# Build SELinux packages for Arch linux in Docker/Podman container
+
 # Usage:
-#    sudo docker build -t arch-selinux-build .
-#
-# or with Podman on Arch Linux in July 2020:
-#    podman --cgroup-manager=cgroupfs build -t arch-selinux-build .
+#    docker build -t arch-selinux-build .
```

Comment:
No. Please keep `sudo` here or document "running from `root`", as running `docker` directly from a non-root user is not a good security practice (it enables a very simple privilege escalation using `docker run -v /:/host -ti archlinux bash`)

Reply:
You are correct here, and it seems I have misunderstood the best practices I have looked at. I was well aware of a user having access to docker has essentially root on a system.

Namely I misinterpreted the 'run containers as non-root' -practice, which actually refers to the user working inside the container, not the user running docker itself.